docs/source/dev-guide/plugins/reporter_backends.rst

```diff
@@ -0,0 +1,162 @@
+=================
+Reporter Backends
+=================
+
+A reporter backend is a plugin hook that allows you to change the output displayed
+for existing conda commands. Internally, conda uses this to differentiate between
+displaying normal, "console" output or displaying "json" output. An example of each
+of these reporter backend types is shown below:
```

Comment:
Should there be a couple of example code blocks after this paragraph?

Reply:
Thanks for catching this. This paragraph shouldn't actually be there and was something I forgot to update when last working with the code. I will replace it with something more suitable.